Sources/Orbit/Components/Stepper.swift

```diff
@@ -0,0 +1,262 @@
+import SwiftUI
+
+/// Enables incremental changes of a counter without a direct input.
+///
+/// - Note: [Orbit definition](https://orbit.kiwi/components/interaction/stepper/)
+public struct Stepper: View {
+
+    @Binding var value: Int
+
+    let minValue: Int
+    let maxValue: Int
+    let isEnabled: Bool
+    let style: Style
+
+    @Environment(\.colorScheme) var colorScheme
+    
+    @State var isIncreased = false
+
+    public var body: some View {
+        HStack(alignment: .center, spacing: 0) {
+            decrementButton
+                .accessibility(addTraits: [.isButton])
+
+            valueContainer
+                .frame(minWidth: 40)
+                .accessibility(value: .init("\(value)"))
+
+            incrementButton
+                .accessibility(addTraits: [.isButton])
+        }
+    }
+
+    @ViewBuilder var valueContainer: some View {
+        Text("\(value)")
+            .foregroundColor(colorScheme == .light ? .cloudDark : .cloudLight)
+            .transition(
+                AnyTransition.asymmetric(
+                    insertion: .move(edge: isIncreased ? .top : .bottom),
+                    removal: .move(edge: isIncreased ? .bottom : .top)
+                )
+                .combined(with: .opacity)
+            )
+            .id("\(value)")
+    }
+
+    @ViewBuilder var decrementButton: StepperButton {
+        StepperButton(
+            isIncrement: false,
+            isEnabled: isEnabled ? value > minValue : false,
+            style: style,
+            clickAction: {
+                isIncreased = false
+
+                withAnimation {
+                    value -= 1
+                }
+            }
+        )
+    }
+
+    @ViewBuilder var incrementButton: StepperButton {
```

Comment:
`decrementButton` and `incrementButton` are so similar that I keep trying to find a way to make just one declaration with a few alterations 😅 

- `isEnabled` should use environment; the checks around min/max values could be unified (possibly even using a range)
- `value` changes could use a ternary operator

Not sure what to do about `isIncreased` though 🤔

Reply:
We don't need `isIncreased` anymore since we got rid of animation